promgen/static/js/promgen.vue.js

```diff
@@ -67,16 +63,14 @@ const app = Vue.createApp({
         },
         setSilenceLabels(labels) {
             silenceStore.setLabels(labels);
-            silenceStore.showForm();
-            scroll(0, 0);
+            this.showModal('silenceModal');
```

Comment:
GitHub does not allow me to make code suggestions if there are deleted lines in your changes, so I will write the desired result here:

```
         setSilenceLabels(labels) {
             silenceStore.setLabels(labels);
             silenceStore.showModal();
         },
```

This will trigger the modal show via the silence store.